File: elixir/7ku/love_vs_friendship.ex
defmodule Kata do
  def words_to_marks(s) do
    s
    |> String.graphemes()
    |> Enum.map(fn x ->
        String.to_charlist(x) |> Enum.reduce(0, &(&1 + &2 - 96))
    end)
    |> Enum.sum
  end
end

File: ruby/7ku/Love vs friendship.rb
# frozen_string_literal: true
# Love vs friendship
# https://www.codewars.com/kata/59706036f6e5d1e22d000016/train/ruby

def words_to_marks(string)
  string.chars.sum { |char| char.ord - 96 }
end

def words_to_marks(string)
  string.sum - string.size * 96
end

p words_to_marks("selfness")